warplib/src/warp_memory.cairo

```diff
@@ -34,4 +41,45 @@ impl WarpMemoryImpl of MemoryTrait {
     fn append(ref self: WarpMemory, value: felt252) {
         self.insert(self.pointer, value);
     }
+
+    fn store_raw_array(ref self: WarpMemory, ref value: Array::<felt252>) {
```

Comment:
perhaps a `store_raw` name

Reply:
I would like to adhere to your naming convention so `write_cells` or `write_multiple` will be bettter. It'd reflect that it is a util function same as `write`, operating on the same level, but for multiple cells